makefile

```diff
@@ -35,8 +35,18 @@ omim.owl: omim.ttl mondo_exactmatch_omim.sssom.owl mondo_exactmatch_omimps.sssom
 		query --update sparql/hgnc_links.ru \
 		convert -f ofn -o $@
 
-mondo_genes.csv: omim.owl
+mondo_genes.robot.tsv: omim.owl
```

Comment:
Update: Output in TSV now instead of CSV
- ROBOT automatically does this based on the file extension

Reply:
Now renamed to `mondo-omim-genes.robot.tsv`